src/telemetry/metrics.yaml

```diff
@@ -1776,3 +1761,73 @@ session:
       - mcleinman@mozilla.com
       - vpn-telemetry@mozilla.com
     expires: never
+
+  daemon_conn_health_no_signal:
```

Comment:
Creating these new metrics would have the same effect as sending the same metric is multiple pings, it just makes the metric design a bit repetitive.